### DISCUSSION TITLE
客户端的登录是有什么作用的

### DISCUSSION THREAD

**Discussion by EuropeJing** [rustdesk/rustdesk#916] [Q&A]
你好，我想问下客户端的登录是有什么用的，又找不到地方注册，也没找到说明。点确定后出现的网址又API，API又没有找到资料。能不能问下这个要怎么用？
![image](https://user-images.githubusercontent.com/25347249/177111712-fcb377f8-74dd-4969-a9e9-82676504af25.png)

**Comment by supply9243** [comment]
https://github.com/rustdesk/rustdesk/wiki/FAQ#:~:text=the%20time%20being-,Change,-id%2C%20addressbook/api
